models/intel/text-recognition-0013/description/text-recognition-0013.md

```diff
@@ -0,0 +1,53 @@
+# text-recognition-0013
+
+## Use Case and High-Level Description
+
+This is a network for text recognition scenario. It consists of VGG16-like backbone and bidirectional LSTM encoder-decoder.
+The network is able to recognize case-insensitive alphanumeric text (36 unique symbols).
+
+## Example
+
+![](./openvino.jpg) -> openvino
+
+## Specification
+
+| Metric                                         | Value              |
+| ---------------------------------------------- | ------------------ |
+| Accuracy on the alphanumeric subset of ICDAR13 | 0.8739             |
+| Text location requirements                     | Tight aligned crop |
+| GFlops                                         | 1.488              |
+| MParams                                        | 5.567              |
+| Source framework                               | PyTorch            |
+
+## Inputs
+
+Shape: [1x1x32x120] - An input image in the format [BxCxHxW],
+where:
+  - B - batch size
+  - C - number of channels
+  - H - image height
+  - W - image width
+
+Note that the source image should be tight aligned crop with detected text converted to grayscale.
+
+## Outputs
+The net outputs a blob with the shape [30, 1, 37] in the format [WxBxL],
```

Comment:
as you have the issue with additional outputs, I think it will be useful if you add name for output in description (and for input too, for alignment)

Reply:
Added info about inputs and outputs